tools/accuracy_checker/accuracy_checker/annotation_converters/fewrel.py

```diff
@@ -0,0 +1,389 @@
+                                                                      h[2]:t[1]] + "$ " + t_name + " $" + ex_text_a[
+                                                                                                          t[2]:]
+            else:
+                ex_text_a = ex_text_a[:t[1]] + "$ " + t_name + " $" + ex_text_a[
+                                                                      t[2]:h[1]] + "# " + h_name + " #" + ex_text_a[
+                                                                                                          h[2]:]
+
+            if h[1] < t[1]:
+                h[1] += 2
+                h[2] += 2  # word in h[1]:h[2] positions moves right by "#($) " or "$(#) " (2 symbols)
+                t[1] += 6
+                t[2] += 6  # word in t[1]:t[2] positions moves right by "#($) " * 3 (6 symbols)
+            else:
+                h[1] += 6
+                h[2] += 6
+                t[1] += 2
+                t[2] += 2
+
+            tokens_a = self.tokenizer.tokenize(ex_text_a)
+
+            if len(tokens_a) > self.max_seq_length - 2:
+                tokens_a = tokens_a[:(self.max_seq_length - 2)]
+
+            tokens = ["[CLS]"] + tokens_a + ["[SEP]"]
+            # ents = ["UNK"] + entities_a + ["UNK"]
+            segment_ids = [0] * len(tokens)
+
+
+            input_ids = self.tokenizer.convert_tokens_to_ids(tokens)
+
+            # The mask has 1 for real tokens and 0 for padding tokens. Only real
+            # tokens are attended to.
+            input_mask = [1] * len(input_ids)
+
+            # Zero-pad up to the sequence length.
+            padding = [0] * (self.max_seq_length - len(input_ids))
+            # padding_ = [-1] * (self.max_seq_length - len(input_ids))
+            input_ids += padding
+            input_mask += padding
+            segment_ids += padding
+
+                # add index to make identifier unique
+            identifier = ['input_ids_{}'.format(example_index),
+                          'input_mask_{}'.format(example_index),
+                          'segment_ids_{}'.format(example_index),
+                          'input_ent_{}'.format(example_index),
+                          'ent_mask_{}'.format(example_index),
+                          'label_ids_{}'.format(example_index),
+                        ]
+
+            annotation = RelationClassificationAnnotation(
+                identifier,
+                np.array(label_map[example['label']]),
+                np.array(input_ids),
+                np.array(input_mask),
+                np.array(segment_ids),
+                input_ent[example_index],
+                ent_mask[example_index].reshape(self.max_seq_length),
+                np.array(label_map[example['label']]),
+                tokens
+            )
+            annotations.append(annotation)
+            unique_id += 1
+        return ConverterReturn(annotations, None, None)
+
+    # @staticmethod
```

Comment:
is it commented as non-working and in progress or just reductant code, which should be removed?

Reply:
Commented code is rudiment from squad.py, run_fewrel.py and should be removed